docs/standard/io/buffers.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "System.Buffers - .NET"
+ms.date: "10/15/2019"
+ms.technology: dotnet-standard
+helpviewer_keywords: 
+  - "buffers [.NET]"
+  - "I/O [.NET], buffers"
+author: rick-anderson
+ms.author: riande
+---
+# Work with Buffers in .NET
+
+Review questions.
+
+Can I remove the `// Return the position.` comment?
+
+```Captain_Obvious
+// Return the position.
+   return reader.Position;
+```
```

Comment:
This needs a real intro no?

Reply:
No, it's an embedded review question. They always ignore review questions so I put it at the top of the doc,

Reply:
I can remove that and PR the actual code.

Reply:
whatever works 😄 but you still need an intro of what buffers are and what problems they solve before you start going into details about each main API. @davidfowl is there any info somewhere that describes what buffers are in a high level way (blog, spec, etc)?